datamol/reactions/_reactions.py

```diff
@@ -164,7 +164,7 @@ def is_reaction_ok(rxn: dm.ChemicalReaction, enable_logs: bool = False) -> bool:
 def select_reaction_output(
     product: Sequence[Sequence[dm.Mol]],
     product_index: Optional[Union[int, list]] = None,
-    single_output: bool = True,
+    single_products: bool = True,
```

Comment:
@zhu0619 Sorry to be picky here, but can you rename `single_products` to ` single_product`?

Reply:
Oh you're right Lu, it's my bad. But I also prefer `single_product_group` since it's less confusing.

Reply:
I just commit the name refactor.  My local test are all green so I skipped CI for this minor change. 
Let me know if I can merge or feel free to merge it.

Reply:
Thanks Lu. I will merge and release!